learning/katas/go/Core Transforms/Flatten/Flatten/task.md

```diff
@@ -0,0 +1,38 @@
+<!--
+  ~  Licensed to the Apache Software Foundation (ASF) under one
+  ~  or more contributor license agreements.  See the NOTICE file
+  ~  distributed with this work for additional information
+  ~  regarding copyright ownership.  The ASF licenses this file
+  ~  to you under the Apache License, Version 2.0 (the
+  ~  "License"); you may not use this file except in compliance
+  ~  with the License.  You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~  Unless required by applicable law or agreed to in writing, software
+  ~  distributed under the License is distributed on an "AS IS" BASIS,
+  ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~  See the License for the specific language governing permissions and
+  ~  limitations under the License.
+  -->
+
+Flatten
+-------
+
+Flatten is a Beam transform for PCollection objects that store the same data type. Flatten merges 
+multiple PCollection objects into a single logical PCollection.
+
+**Kata:** Implement a 
```

Comment:
Grammatical Nit: Flatten is a primitive transform in the beam model (it can't be efficiently implemented in terms of the other primitives), so one never "implements" a Flatten. One can use a flatten, but I fear saying "implements" might cause learners to head down the wrong path.  (If that's the pedagogical intent here, then no worries).

Reply:
The use of "implement" here means to implement a solution. The same description is used across many task descriptions.